cdk/lib/frontend.ts

```diff
@@ -18,5 +34,360 @@ export class Frontend extends GuStack {
+        PROD: "arn:aws:dynamodb:*:*:table/redemption-codes-PROD",
+      },
+    });
+
+    const policies = [
+      // TODO: can we 'standardise' the way we load config to use the default permissons from GuEc2App?
+      new GuAllowPolicy(this, "SSMGet", {
+        actions: ["ssm:GetParametersByPath"],
+        resources: [
+          `arn:aws:ssm:${this.region}:${this.account}:parameter/${this.stack}/${app}/${this.stage}`,
+        ],
+      }),
+      new GuGetS3ObjectsPolicy(this, "PrivateBucket", {
+        bucketName: "gu-zuora-catalog",
+        paths: [
+          "PROD/Zuora-PROD/catalog.json",
+          "PROD/Zuora-UAT/catalog.json",
+          "PROD/Zuora-DEV/catalog.json",
+        ],
+      }),
+      new GuGetS3ObjectsPolicy(this, "SettingsBucket", {
+        bucketName: "support-admin-console",
+        paths: [`${this.stage}/*`],
+      }),
+      new GuPutCloudwatchMetricsPolicy(this),
+      // TODO: should we move logs to kinesis?
+      new GuAllowPolicy(this, "CloudwatchMetrics", {
+        actions: [
+          "logs:CreateLogGroup",
+          "logs:CreateLogStream",
+          "logs:PutLogEvents",
+          "logs:DescribeLogStreams",
+        ],
+        resources: ["arn:aws:logs:*:*:*"],
+      }),
+      new GuAllowPolicy(this, "StateMachines", {
+        actions: [
+          "states:ListStateMachines",
+          "states:StartExecution",
+          "states:GetExecutionHistory",
+          "states:DescribeStateMachine",
+        ],
+        resources: ["arn:aws:states:*:*:*"],
+      }),
+      new GuAllowPolicy(this, "DynamoPromotions", {
+        actions: [
+          "dynamodb:GetItem",
+          "dynamodb:Scan",
+          "dynamodb:Query",
+          "dynamodb:DescribeTable",
+        ],
+        resources: [
+          membershipSubPromotionsTable,
+          "arn:aws:dynamodb:*:*:table/MembershipSub-Promotions-UAT",
+        ],
+      }),
+      new GuAllowPolicy(this, "DynamoRedemptions", {
+        actions: ["dynamodb:GetItem"],
+        resources: [
+          redemptionCodesTable,
+          "arn:aws:dynamodb:*:*:table/redemption-codes-UAT",
+        ],
+      }),
+      new GuAllowPolicy(this, "StripeSetupIntentLambda", {
+        actions: ["lambda:InvokeFunction"],
+        resources: [
+          `arn:aws:lambda:eu-west-1:${this.account}:function:stripe-intent-${this.stage}`,
+        ],
+      }),
+    ];
+
+    const ec2App = new GuEc2App(this, {
+      applicationPort: 9000,
+      app: "frontend",
+      access: { scope: AccessScope.PUBLIC },
+      certificateProps: {
+        [Stage.CODE]: {
+          domainName:
+            "support.code.theguardian.com.origin.membership.guardianapis.com",
+          hostedZoneId: "Z1E4V12LQGXFEC",
+        },
+        [Stage.PROD]: {
+          domainName:
+            "support.theguardian.com.origin.membership.guardianapis.com",
+          hostedZoneId: "Z1E4V12LQGXFEC",
+        },
+      },
+      monitoringConfiguration: {
+        noMonitoring: true,
```

Comment:
At the moment we are opting out of the 'standard' alarms that the pattern wants to create; we might be able to use some alarms provided via the pattern to reduce the number of custom alarms defined here.